example/pubspec.lock

```diff
@@ -385,7 +385,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "1.0.3"
+    version: "1.0.4"
```

Comment:
I think it makes more sense to make the version "1.1.0" since this is a breaking change 🤔